index.js

```diff
@@ -88,6 +86,15 @@ function wrapFunctions(target, methodNames) {
   });
 }
 
+function _actualMethodKeys(target) {
+  return Object.getOwnPropertyNames(target)
+    .filter(key => {
+      var propertyDescriptor = Object.getOwnPropertyDescriptor(target, key);
+      return propertyDescriptor.get == null && propertyDescriptor.set == null;
```

Comment:
Mind changing this to `return !propertyDescriptor.get && !propertyDescriptor.set;` ?